src/content/1/en/part1b.md

```diff
@@ -386,12 +386,12 @@ const arto = {
   education: 'PhD',
   // highlight-start
   greet: function() {
-    console.log('hello, my name is', this.name)
+    console.log('hello, my name is ' + this.name)
```

Comment:
imo this example is easier to understand for beginners this way, as opposed to logging `this.name` as a second param to `console.log`, as the result that we see in the comments is the actual result the user will see if they try it out for themselves